src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -65,209 +71,238 @@ private void captureOs() {
+                        buildScan.value("CI build number", value));
+                envVariable("bamboo_planName").ifPresent(value ->
+                        addCustomValueAndSearchLink(buildScan, "CI plan", value));
+                envVariable("bamboo_buildPlanName").ifPresent(value ->
+                        addCustomValueAndSearchLink(buildScan, "CI build plan", value));
+                envVariable("bamboo_agentId").ifPresent(value ->
+                        addCustomValueAndSearchLink(buildScan, "CI agent", value));
             }
-            envVariable("GITHUB_WORKFLOW").ifPresent(value ->
-                addCustomValueAndSearchLink("CI workflow", value));
-            envVariable("GITHUB_RUN_ID").ifPresent(value ->
-                addCustomValueAndSearchLink("CI run", value));
-        }
 
-        if (isGitLab()) {
-            envVariable("CI_JOB_URL").ifPresent(url ->
-                buildScan.link("GitLab build", url));
-            envVariable("CI_PIPELINE_URL").ifPresent(url ->
-                buildScan.link("GitLab pipeline", url));
-            envVariable("CI_JOB_NAME").ifPresent(value1 ->
-                addCustomValueAndSearchLink("CI job", value1));
-            envVariable("CI_JOB_STAGE").ifPresent(value ->
-                addCustomValueAndSearchLink("CI stage", value));
-        }
+            if (isGitHubActions()) {
+                Optional<String> gitHubRepository = envVariable("GITHUB_REPOSITORY");
+                Optional<String> gitHubRunId = envVariable("GITHUB_RUN_ID");
+                if (gitHubRepository.isPresent() && gitHubRunId.isPresent()) {
+                    buildScan.link("GitHub Actions build", "https://github.com/" + gitHubRepository.get() + "/actions/runs/" + gitHubRunId.get());
+                }
+                envVariable("GITHUB_WORKFLOW").ifPresent(value ->
+                        addCustomValueAndSearchLink(buildScan, "CI workflow", value));
+                envVariable("GITHUB_RUN_ID").ifPresent(value ->
+                        addCustomValueAndSearchLink(buildScan, "CI run", value));
+            }
 
-        if (isTravis()) {
-            envVariable("TRAVIS_BUILD_WEB_URL").ifPresent(url ->
-                buildScan.link("Travis build", url));
-            envVariable("TRAVIS_BUILD_NUMBER").ifPresent(value ->
-                buildScan.value("CI build number", value));
-            envVariable("TRAVIS_JOB_NAME").ifPresent(value ->
-                addCustomValueAndSearchLink("CI job", value));
-            envVariable("TRAVIS_EVENT_TYPE").ifPresent(buildScan::tag);
-        }
+            if (isGitLab()) {
+                envVariable("CI_JOB_URL").ifPresent(url ->
+                        buildScan.link("GitLab build", url));
+                envVariable("CI_PIPELINE_URL").ifPresent(url ->
+                        buildScan.link("GitLab pipeline", url));
+                envVariable("CI_JOB_NAME").ifPresent(value1 ->
```

Comment:
I know this was in the previous diff as well, but is there a reason this is named `value1` here?

Reply:
No good reason, I updated it 👍